pkg/kstatus/status/apply_status.go

```diff
@@ -0,0 +1,41 @@
+package status
+
+const (
```

Comment:
Recommend replacing all these strings with enums. You can use go-stringer to impl the String() methods and then optionally provide short string values in comments.

Ex: https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/apply/event/event.go#L106

Reply:
Done.